docs/UserGuide.md

```diff
@@ -234,8 +245,8 @@ Additional Details:
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list.
 * **At least one of the optional fields must be provided**.
 * Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without specifying any tags after it.
+* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative. 
```

Comment:
Will it be better to say the following:
"When editing tags, the inputted tag will completely replace the current tag."

Reply:
OK! Thanks yes its more clear